File: Sobczal1.KickBets.Domain/Football/Statistics.cs
namespace Sobczal1.KickBets.Domain.Football;

public class Statistics : BaseDomainEntity
{
    public virtual Fixture Fixture { get; set; } = null!;
    public int? ShotsOnGoal { get; set; }
    public int? ShotsOffGoal { get; set; }
    public int? ShotsInsideBox { get; set; }
    public int? ShotsOutsideBox { get; set; }
    public int? TotalShots { get; set; }
    public int? BlockedShots { get; set; }
    public int? Fouls { get; set; }
    public int? CornerKicks { get; set; }
    public int? Offsides { get; set; }
    public double? Possession { get; set; }
    public int? YellowCards { get; set; }
    public int? RedCards { get; set; }
    public int? GoalkeeperSaves { get; set; }
    public int? TotalPasses { get; set; }
    public int? AccuratePasses { get; set; }
    public double? Passes { get; set; }
}

File: client-app/src/app/models/football/statistics.ts
import {BaseDto} from "../common/base";

export interface StatisticDto extends BaseDto {
    shotsOnGoal: number | null;
    shotsOffGoal: number | null;
    shotsInsideBox: number | null;
    shotsOutsideBox: number | null;
    totalShots: number | null;
    blockedShots: number | null;
    fouls: number | null;
    cornerKicks: number | null;
    offsides: number | null;
    possession: number | null;
    yellowCards: number | null;
    redCards: number | null;
    goalkeeperSaves: number | null;
    totalPasses: number | null;
    accuratePasses: number | null;
    passes: number | null;
}